cpp/src/lists/contains.cu

```diff
@@ -255,14 +255,14 @@ void index_of_nested_types(InputIterator input_it,
   // returned to keep the result `table_view` valid.
   [[maybe_unused]] auto const [keys_tview, unused_column] =
```

Comment:
Looks like these captures are now completely unused?

Reply:
`keys_tview` will be used at line 267. The other variable will never be used but needs to be alive.

Reply:
Sorry, I misread the `[]` as a capture list, but it's actually structured bindings. (Missed the `=`)